fastlane/Fastfile

```diff
@@ -829,15 +824,36 @@ end
       type: "appstore",
       team_id: get_required_env("EXT_EXPORT_TEAM_ID"),
       readonly: options[:readonly] || is_ci,
-      app_identifier: [
-        "com.codality.NotationalFlow",
-        "com.codality.NotationalFlow.Share",
-        "com.codality.NotationalFlow.Widgets",
-      ],
+      app_identifier: simplenote_app_identifiers,
       template_name: "NotationalFlow Keychain Access (Distribution)",
     )
   end
 
+  # Compiles the array of bundle identifiers for the different targets that
+  # make up the Simplenote app, to be used as the `app_identifier` parameter
+  # for the `match` action.
+  def simplenote_app_identifiers(root_bundle_id: APP_STORE_BUNDLE_IDENTIFIER)
+    extension_bundle_ids =
+      %w[Share Widgets Intents]
+      .map { |suffix| "#{root_bundle_id}.#{suffix}" }
+
+    [root_bundle_id, *extension_bundle_ids]
```

Comment:
Super-nit: I'm not sure what rubocop would say, but iirc the ruby conventions would tell to not let the `=` hanging at the end of the line, but instead write it this way:

```ruby
    extension_bundle_ids = %w[Share Widgets Intents]
      .map { |suffix| "#{root_bundle_id}.#{suffix}" }
```
Or alternatively this way:
```ruby
    extension_bundle_ids = %w[Share Widgets Intents].map do |suffix|
      "#{root_bundle_id}.#{suffix}"
    end
```

Not a big deal, only style, just thought I'd mention fyi.

Alternative idea, if we want to be very nitpicky (personally I like it even better, but it's a matter of taste, so take it or leave it):
```ruby
  def simplenote_app_identifiers(root_bundle_id: APP_STORE_BUNDLE_IDENTIFIER)
    [nil, 'Share', 'Widgets', 'Intents'].map do |suffix|
      [root_bundle_id, suffix].compact.join('.')
    end
  end
```